index.html

```diff
@@ -0,0 +1,295 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="./css/main.css">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU"
+    crossorigin="anonymous">
+    <link rel="shortcut icon" href="favicon.ico" type="image/x-icon">
+  <link rel="icon" href="favicon.ico" type="image/x-icon">
+  <title>Free Videos</title>
+
+</head>
+
+<body>
+  <section class="full-container banner">
+    <video autoplay muted loop class="myVideo">
+      <source src="./assets/videos/video-loop-1.mp4" type="video/mp4">
+    </video>
+    <div class="navbar">
+      <nav class="navbar-nav">
+        <ul>
+          <li class="nav-items">
+            <a href="#">License</a>
+          </li>
+          <li class="nav-items">
+            <a href="#">Popular Videos</a>
+          </li>
+          <li class="nav-items">
+            <a href="#">Free Photos</a>
+          </li>
+          <li class="nav-items">
+            <a href="#">
+              <i class="fas fa-ellipsis-h"></i>
+            </a>
+          </li>
+          <li class="nav-items menu-clicker">
+            <a href="#">
+              <i class="fas fa-bars"></i>
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </div>
+    <article class="header-center">
+      <img src="./assets/images/pexels-logo.png" alt="pexels">
+      <p> Completely free stock videos.
+        <a href="#" class="learn-more">Learn More</a>
+      </p>
+      <form action="submit" method="POST">
+        <input type="text" placeholder="Search for free video">
```

Comment:
You can have the `input` without the form. 
Also, the `action="submit"` is being interpreted as if submit was a file. So when you press the `enter` key after filling the form the browser throws an error saying the file is not found. I'm NOT asking to implement a search function. Just fix the input so that it does nothing when you press the `enter` key

Reply:
fixed